cosmic-settings/src/pages/desktop/workspaces.rs

```diff
@@ -136,14 +135,14 @@ fn multi_behavior() -> Section<crate::pages::Message> {
             let descriptions = &section.descriptions;
             settings::view_section(&section.title)
                 .add(settings::item_row(vec![radio(
-                    &descriptions[span],
+                    text(&descriptions[span]),
```

Comment:
Make sure to use `text::body` instead

Reply:
Oh, I've just noticed that workspace orientation didn't need changes. Should I maybe put just &*descriptions instead of using text?

Reply:
Sure

Reply:
Sorry for the many questions, but a few places in touchpad.rs have `text(&description[])`: should that be changed to `&*description[]`, or would it be better for everything to use text::body?

Reply:
It's generally best to use `text::body` to ensure that the correct text style is set.

Reply:
Should I also do this for other pages (e.g. Mouse), or just these two (and maybe import `text::body` instead of `text` where possible)?

Reply:
If you see any area that omitted it, it would be good to fix it.

Reply:
I think I've caught most things, and nothing seems to be changed visually.